modules/serverless-rn-1-34-0.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-34-0_{context}"]
+= Red Hat {ServerlessProductName} 1.34
+
+{ServerlessProductName} 1.34 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="new-features-1-34-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.13.
+* {ServerlessProductName} now uses Knative Eventing 1.13.
+* {ServerlessProductName} now uses Kourier 1.13.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.13.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.13.
+* The `kn func` CLI plugin now uses `func` 1.15.
+
+* It is now possible to support mutliple configuration for OpenAPI within the same namespace.
+
+* Encryption Serving is now available as a Technology Preview (TP) feature.
+
+* {ServerlessProductName} 1.34 introduces a new feature that allows workflows to access different OpenShift clusters through configuration. This feature enables users to define REST calls within a workflow to seamlessly interact with multiple clusters.
+
+* The Jobs Service liveness checks is now enhanced to limit the time required to retrieve the leader status. A new system property, `kogito.jobs-service.management.leader-check.expiration-in-seconds`, has been introduced, allowing you to configure the maximum time allowed for the leader status check.
+
+* {ServerlessProductName} Serving transport encryption feature allows transporting data over secured and encrypted HTTPS connections using TLS. This is now available as a Technology Preview (TP) feature.
+
+* Go function using S2I builder is now available as a Technology Preview (TP) feature.
+
+* Multi-container support for Knative Serving allows you to use a single Knative service to deploy a multi-container pod. It also supports the the `readiness` and `liveness` probe values for multiple containers.
+
+* The new trigger filters feature is now available as a Technology Preview. These filters are enabled by default and allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
```

Comment:
We already documented the new trigger filters for 1.32. No need to repeat this, IMO. It's not new.

Reply:
Okay, removed!